<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>693</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>add random cards</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="topCardLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bottomCardLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="leftCardLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>141</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="rightCardLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tableCardLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
